integration_tests/helper/helper.go

```diff
@@ -35,3 +55,303 @@ func ReadResponseBody(responseBody *bytes.Buffer) []byte {
+	} `json:"credentials"`
+}
+
+type ServiceHelper struct {
+	InstanceID     string
+	ServiceID      string
+	PlanID         string
+	Catalog        *catalog.ComposeCatalog
+	Cfg            *config.Config
+	Logger         lager.Logger
+	BrokerInstance *broker.Broker
+	BrokerAPI      http.Handler
+	ComposeClient  compose.Client
+}
+
+func (s *ServiceHelper) Bind() (binding *BindingData) {
+	binding = &BindingData{
+		ID:    NewUUID(),
+		AppID: NewUUID(),
+	}
+	resp := DoRequest(s.BrokerAPI, NewRequest(
+		"PUT",
+		fmt.Sprintf("/v2/service_instances/%s/service_bindings/%s", s.InstanceID, binding.ID),
+		strings.NewReader(fmt.Sprintf(`{
+			"service_id": "%s",
+			"plan_id": "%s",
+			"bind_resource": {
+				"app_guid": "%s"
+			},
+			"parameters": "{}"
+		}`, s.ServiceID, s.PlanID, binding.AppID)),
+		s.Cfg.Username,
+		s.Cfg.Password,
+	))
+	Expect(resp.Code).To(Equal(201))
+
+	err := json.NewDecoder(resp.Body).Decode(&binding)
+	Expect(err).ToNot(HaveOccurred())
+
+	Expect(binding.Credentials.URI).NotTo(BeEmpty())
+	Expect(binding.Credentials.Username).NotTo(BeEmpty())
+	Expect(binding.Credentials.Password).NotTo(BeEmpty())
+	Expect(binding.AppID).NotTo(BeEmpty())
+
+	return binding
+}
+
+func (s *ServiceHelper) Unbind(bindingID string) {
+	resp := DoRequest(s.BrokerAPI, NewRequest(
+		"DELETE",
+		fmt.Sprintf("/v2/service_instances/%s/service_bindings/%s", s.InstanceID, bindingID),
+		nil,
+		s.Cfg.Username,
+		s.Cfg.Password,
+		UriParam{Key: "service_id", Value: s.ServiceID},
+		UriParam{Key: "plan_id", Value: s.PlanID},
+	))
+	Expect(resp.Code).To(Equal(200))
+	// Response will be an empty JSON object for future compatibility
+	var data map[string]interface{}
+	err := json.NewDecoder(resp.Body).Decode(&data)
+	Expect(err).ToNot(HaveOccurred())
+}
+
+func (s *ServiceHelper) Deprovision() {
+	resp := DoRequest(s.BrokerAPI, NewRequest(
+		"DELETE",
+		"/v2/service_instances/"+s.InstanceID,
+		nil,
+		s.Cfg.Username,
+		s.Cfg.Password,
+		UriParam{Key: "service_id", Value: s.ServiceID},
+		UriParam{Key: "plan_id", Value: s.PlanID},
+		UriParam{Key: "accepts_incomplete", Value: "true"},
+	))
+	Expect(resp.Code).To(BeEquivalentTo(202))
+
+	var deprovisionResp brokerapi.DeprovisionResponse
+	err := json.NewDecoder(resp.Body).Decode(&deprovisionResp)
+	Expect(err).ToNot(HaveOccurred())
+
+	operationState := PollForOperationCompletion(
+		s.Cfg, s.BrokerAPI, s.InstanceID,
+		s.ServiceID, s.PlanID, deprovisionResp.OperationData,
+	)
+	Expect(operationState).To(Equal("succeeded"), "returns success")
+}
+
+func Provision(serviceID string, planID string) (s *ServiceHelper) {
+	s = &ServiceHelper{
+		InstanceID: NewUUID(),
+		ServiceID:  serviceID,
+		PlanID:     planID,
+		Cfg: &config.Config{
+			Username: randString(10),
+			Password: randString(10),
+			DBPrefix: "test-suite",
+			APIToken: os.Getenv("COMPOSE_ACCESS_TOKEN"),
+		},
+	}
+	Expect(s.Cfg.APIToken).NotTo(BeEmpty(), "Please export $COMPOSE_ACCESS_TOKEN")
```

Comment:
Are you using the `run-all-scripts.sh` script? ... since I think someone made that attempt to pull a key out of your pass repo so maybe it's getting confused?